lib/templates/structure/sidebar-primary.php

```diff
@@ -5,16 +5,19 @@
  * @package Beans\Framework\Templates\Structure
  *
  * @since   1.0.0
+ * @since   1.5.0 Added ID and tabindex for skip links.
  */
 
 beans_open_markup_e(
 	'beans_sidebar_primary',
 	'aside',
 	array(
 		'class'     => 'tm-secondary ' . beans_get_layout_class( 'sidebar_primary' ), // Automatically escaped.
+		'id'        => 'beans-primary-sidebar',
```

Comment:
file is called **sidebar**-_primary_
element is called beans_**sidebar**__primary_
class is called **sidebar**__primary_

but
id is called beans-_primary_-**sidebar**

id name should follow the naming convention, right?

Needs to be updated in the tests too